packages/nuxt/src/pages/module.ts

```diff
@@ -266,36 +263,14 @@ export default defineNuxtModule({
       })
     })
 
-    nuxt.hook('nitro:init', (nitro) => {
-      if (nuxt.options.dev || !nitro.options.static || nuxt.options.router.options.hashMode) { return }
```

Comment:
Is there anything that preserves the `nuxt.options.dev` or `nitro.options.static` guards?

I'm guessing the `.server.ts` extension serves the purpose of `nuxt.options.dev`.